crates/teloxide/examples/db_remember.rs

```diff
@@ -12,7 +12,7 @@ use teloxide::{
 
 type MyDialogue = Dialogue<State, ErasedStorage<State>>;
 type MyStorage = std::sync::Arc<ErasedStorage<State>>;
-type HandlerResult = Result<(), Box<dyn std::error::Error + Send + Sync>>;
+type Result = std::result::Result<(), Box<dyn std::error::Error + Send + Sync>>;
```

Comment:
I don't think this change is necessary, please revert it.

Also a tip: you can use defaults, so that `Result` can still be used normally:
```rust
type Result<T = (), E = Box<dyn std::error::Error + Send + Sync>> = std::result::Result<T, E>;
```

Reply:
Hm, actually, the CI seems to fail here, did you miss some uses of `Result`?..

Could you add a test to the `BotCommands` tests that uses `type Result = ();` and make sure that it passes?

Reply:
Uncomment the line: https://github.com/teloxide/teloxide/blob/b826c3a34be7fb7a30c66d6f0c9e3990bf0d827b/crates/teloxide/Cargo.toml#L76

Reply:
(still, please undo this change and add a test)

Reply:
Done, I didn’t see the tests directory.